securedrop/journalist_templates/_sources_confirmation_final_modal.html

```diff
@@ -0,0 +1,19 @@
+<div id="{{ modal_data.modal_id }}" class="menu-modal-dialog">
+  <a href="#close" class="external"></a>
+  <div id="delete-confirm-menu-dialog">
+    <p>
+      {{ gettext('When the login account for a source is deleted:') }}
+      <ul>
+        <li>{{ gettext('The source will not be able to log in with their codename again') }}</li>
+        <li>{{ gettext('You will not be able to send them replies.') }}</li>
+        <li>{{ gettext('All files and messages from that source will also be destroyed.') }}</li>
+      </ul>
+    </p>
+    <p>
+      <span class="modal-danger-text">{{ gettext('Are you sure this is what you want?') }}</span</p>
+    <p>
+      <a href="#close" id="cancel-collections-deletions" title="{{ gettext('Cancel') }}" class="btn cancel small">{{ gettext('Cancel') }}</a>
+      <button type="submit" id="delete-collections-confirm" name="action" value="delete" class="small danger">{{ gettext('Yes, Delete Selected Source Accounts') }}</button>
```

Comment:
As @emkll observed these two buttons still render at different height for me.

Reply:
:heavy_check_mark: cut the knot and just wrapped them in a flexbox